pkg/features/kube_features.go

```diff
@@ -506,7 +507,7 @@ var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureS
 	ProcMountType:                               {Default: false, PreRelease: utilfeature.Alpha},
 	TTLAfterFinished:                            {Default: false, PreRelease: utilfeature.Alpha},
 	KubeletPodResources:                         {Default: false, PreRelease: utilfeature.Alpha},
-	WindowsGMSA:                                 {Default: false, PreRelease: utilfeature.Alpha},
+	WindowsGMSA:                                 {Default: false, PreRelease: utilfeature.Beta},
```

Comment:
Let's move this out of this PR for now. Once the overall code changes have been reviewed and all desired tests are passing, we can consider the feature bump a little later in v1.15.

Reply:
👍